docs/synapse/userguides/storm_ref_type_specific.ipynb

```diff
@@ -67,6 +68,279 @@
+    "\n",
+    "**Examples:**\n",
+    "\n",
+    "Lift the ``ou:org`` node(s) whose ``:names`` property contains the value ``vertex``:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Define and print test query\n",
+    "q = 'ou:org:names*[=vertex]'\n",
+    "print(q)\n",
+    "# Execute the query to test it and get the packed nodes (podes).\n",
+    "podes = await core.eval(q, num=1, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "Lift the x509 certificate nodes that reference the domain ``microsoft.com``:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true,
+    "hideOutput": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make a node\n",
+    "q = '[crypto:x509:cert=\"*\" :identities:fqdns=(microsoft.com,verisign.com)]'\n",
+    "# Run the query and test\n",
+    "podes = await core.eval(q, num=1, cmdr=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Define and print test query\n",
+    "q = 'crypto:x509:cert:identities:fqdns*[=microsoft.com]'\n",
+    "print(q)\n",
+    "# Execute the query to test it and get the packed nodes (podes).\n",
+    "podes = await core.eval(q, num=1, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "Downselect a set of ``ou:org`` nodes to include only those with a name that starts with \"acme\":"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true,
+    "hideOutput": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make a node\n",
+    "q = '[ou:org=\"*\" :alias=acme :names=(\"Acme Corporation\",\"Acme Service Corporation\",Acme)]'\n",
+    "# Run the query and test\n",
+    "podes = await core.eval(q, num=1, cmdr=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Define and print test query\n",
+    "q = '<ou:orgs> '\n",
+    "q1 = 'ou:org:alias=acme '\n",
+    "q2 = '+:names*[^=acme]'\n",
+    "print(q + q2)\n",
+    "# Execute the query to test it and get the packed nodes (podes).\n",
+    "podes = await core.eval(q1 + q2, num=1, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "Not all comparison operators are supported as part of the array \"by\" syntax. For example, equals ( ``=`` ) and prefix ( ``^=`` ) operators are supported, but other standard comparison operators (e.g., ``>=`` ) and extended comparison operators (e.g., ``~=`` ) are not.\n",
```

Comment:
This operator limitation sounds like a bug?

Reply:
WM-372